engine/access/rest/server.go

```diff
@@ -13,8 +13,26 @@ import (
 	"github.com/onflow/flow-go/module"
 )
 
+const (
+	// DefaultReadTimeout is the default read timeout for the HTTP server
+	DefaultReadTimeout = time.Second * 15
+
+	// DefaultWriteTimeout is the default write timeout for the HTTP server
+	DefaultWriteTimeout = time.Second * 30
+
+	// DefaultIdleTimeout is the default idle timeout for the HTTP server
+	DefaultIdleTimeout = time.Second * 60
```

Comment:
maybe the Idle timeout should be even bigger?

Reply:
it's configurable now, so operators can choose